### DISCUSSION TITLE
a spread cluster identified by running findclusters

### DISCUSSION THREAD

**Discussion by pengxin2019** [stuart-lab/signac#822] [Q&A] [answered]
Hi Tim,
Sorry for having so many questions.
I have a cluster which is pretty spread across everywhere whatever resolution I use:
When I use resolution 0.8, here is what all clusters look like:
[integrated.clusterspeaks_snn_res.0.8.pdf](https://github.com/timoast/signac/files/7339901/integrated.clusterspeaks_snn_res.0.8.pdf)
This is what cluster 18 looks like:
[integrated.0.8.cluster18.pdf](https://github.com/timoast/signac/files/7339903/integrated.0.8.cluster18.pdf)

actually when I use another resolution, this cluster 18 is still there.

Here are my codes producing plots above:
```
integrated <- RunUMAP(object = integrated, reduction = 'integrated_lsi', dims = 2:30)
integrated <- FindNeighbors(object = integrated, reduction = 'integrated_lsi', dims = 2:30, do.plot = TRUE)
integrated.0.8 <- FindClusters(object = integrated, verbose = FALSE, algorithm = 3, resolution = 0.8) 

```
I do not have a reason to exclude cluster 18 from my integrated object since they have passed quality control anyway.

Can you give me some suggestions?
Thanks
Best

**Comment by timoast** [comment, marked as answer]
Since the clustering is performed in higher dimensions, the cluster membership of cells will not always visually align with the position of cells on the 2-dimensional UMAP embedding. Often when you see cells belonging to one cluster scattered throughout the embedding, it is an indication that they may be low-quality or a technical artefact. I would recommend revisiting the QC metrics you have calculated for these cells.
